src/main/scala/io/sdkman/changelogs/InfrastructorMigrations.scala

```diff
@@ -0,0 +1,29 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "010")
+class InfrastructorMigrations {
+
+  @ChangeSet(order = "001", id = "001_add_infrastructor_candidate", author = "stanislav.tiurikov")
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "infrastructor",
+      name = "Infrastructor",
+      description = "Infrastructor is an open source server provisioning tool written in Groovy",
+      websiteUrl = "http://infrastructor.io/").insert()
+  }
+
+  @ChangeSet(order = "002", id = "002_add_infrastructor_0.1.8", author = "stanislav.tiurikov")
```

Comment:
I would rather not insert the first version as part of the migration since we want to test that you are able to use the API for the initial release. So line 18-27 can be dropped.